### PR TITLE
adding streaming webassembly features

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

File: javascript/builtins/WebAssembly.json [hunk 1]
```diff
@@ -1380,6 +1380,64 @@
...
+              },
+              "edge_mobile": {
+                "version_added": true,
+                "flag": {
+                  "type": "preference",
+                  "name": "Experimental JavaScript Features"
+                }
+              },
+              "firefox": {
+                "version_added": "58",
```
Comment (Elchi3) [inline javascript/builtins/WebAssembly.json]: No trailing comma here

File: javascript/builtins/WebAssembly.json [hunk 2]
```diff
@@ -1380,6 +1380,64 @@
...
+              },
+              "firefox_android": {
+                "version_added": "58",
```
Comment (Elchi3) [inline javascript/builtins/WebAssembly.json]: and here

File: javascript/builtins/WebAssembly.json [hunk 3]
```diff
@@ -1440,6 +1498,64 @@
...
+              },
+              "edge_mobile": {
+                "version_added": true,
+                "flag": {
+                  "type": "preference",
+                  "name": "Experimental JavaScript Features"
+                }
+              },
+              "firefox": {
+                "version_added": "58",
```
Comment (Elchi3) [inline javascript/builtins/WebAssembly.json]: and here

File: javascript/builtins/WebAssembly.json [hunk 4]
```diff
@@ -1440,6 +1498,64 @@
...
+              },
+              "firefox_android": {
+                "version_added": "58",
```
Comment (Elchi3) [inline javascript/builtins/WebAssembly.json]: and here

**Comment by chrisdavidmills** [comment]
dammnit, that was a proper schoolboy error! I have fixed them, and I am enjoying the musical accompaniment ;-)

While writing my kids' Raspberry Pi course recently, I have been working in Python a lot, and trailing slashes just make so much sense. Silly JS.

**Pull request merged**